bigtable/admin.go

```diff
@@ -432,6 +474,33 @@ func (ac *AdminClient) updateTableWithConf(ctx context.Context, conf *UpdateTabl
 		}
 	}
 
+	if conf.automatedBackupPolicy != nil {
```

Comment:
nil semantic here differ from nil semnatics of the conf above

Reply:
Thanks. Updated (discussed offline)